wonderland.js

```diff
@@ -27,7 +31,28 @@ export class Wonderland extends Scene {
                 specularity: 0.5,
                 texture: new Texture("assets/stars.png"),
             }),
+            dali: new Material(new defs.Textured_Phong(1), {
+                color: color(0, 0, 0, 1),
+                ambient: 1, diffusivity: 0, specularity: 0,
+                texture: new Texture("assets/dali.jpg", "LINEAR_MIPMAP_LINEAR")
+            }),
+            obelisk_tip: new Material(new defs.Phong_Shader(), {
+                ambient: 0.8,
+                diffusivity: 0.34615,
+                specularity: 0.95,
+                color: color(0.83, 0.67, 0.22, 1),
+            })
         };
+
+        // STUPID WAY OF DOING THIS
+        this.shapes.obelisk_base.arrays.texture_coord = [
+            Vector.of(0, 0), Vector.of(1, 0), Vector.of(0, 1), Vector.of(1, 1), // bottom face
+            Vector.of(0, 0), Vector.of(1, 0), Vector.of(0, 1), Vector.of(1, 1), // top face
+            Vector.of(0, 0), Vector.of(1, 0), Vector.of(0, 8), Vector.of(1, 8),
+            Vector.of(0, 0), Vector.of(1, 0), Vector.of(0, 8), Vector.of(1, 8),
+            Vector.of(0, 0), Vector.of(1, 0), Vector.of(0, 8), Vector.of(1, 8),
+            Vector.of(0, 0), Vector.of(1, 0), Vector.of(0, 8), Vector.of(1, 8),
+        ]
```

Comment:
Ya what's the alternative to this? You're trying to prevent the image from stretching, right?

Reply:
Oh there's just a quick way of doing it with one line if it's the same for all 6 faces, but this is actually the only way for doing this, bc i need 2 of the faces to have different texture than the other 4.

Reply:
Just removed unused imports!